### DISCUSSION TITLE
How to modify source code

### DISCUSSION THREAD

**Discussion by gbertolottiKS** [nextcloud/all-in-one#5334] [Questions] [answered]
Hi.
We would like to test the changes made in https://github.com/nextcloud/all-in-one/pull/5132: what's the best way to achieve it?
Can we directly modify the files changed in the PR above?
We have a test environment where Nextcloud AIO is already installed and running.

Sorry for the dumb question,
thanks

**Comment by szaimen** [comment, marked as answer]
Hi, you cannot really modify the source code - at least not persistently. However you could exec into the mastercontainer and then apply the changes via git apply. Please note that the PR is not complete yet which is why it was not merged.
